draft-mouris-cfrg-mastic.md

```diff
@@ -1266,14 +1301,25 @@ security analysis of Mastic is provided in {{MPDST24}}.
 
 # IANA Considerations
 
-TODO
+IANA is requested to add a new identifier to the "Verifiable Distributed Aggregation Functions (VDAF)" registry, that is described in {{Section 10 of !VDAF}}. In short, VDAF identifiers are used for domain separation, which guards against failures of entropy sources, by ensuring that invocations of different VDAFs use different derived values, even if they are invoked with the same underlying random data.
+
+The new requested entry to the VDAF Identifier registry is as follows:
```

Comment:
nit: A little more concise of an instruction for IANA would be the following (avoids giving the registry more than one name):
```suggestion
The new requested entry to the registry is as follows:
```

Reply:
Oops, forgot this is what we called it in the base draft. Disregard.